objcgen/extensions.cs

```diff
@@ -134,4 +134,15 @@ public static void IncrementValue<TKey> (this Dictionary<TKey, int> dictionary,
 				dictionary[key] = new List<TListValue> () { val };
 		}
 	}
+
+	public static class ListExtensions {
+		public static bool ContainsType (this List<ProcessedType> list, Type type)
```

Comment:
I do not love this, but since we query it in many places it seemed the least bad option.